genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name, id = Random.rand(1..1_000))
+    @id = id
+    @name = name
+    @items = []
+  end
```

Comment:
@PraisesPJMT in this constructor, how about refactoring it as follows:
def initialize(name)
    super(id = Random.rand(1..1000))
    @id = id
    @name = name
    @items = []
  end

Reply:
Thanks, @benardop for reviewing my work!
Suggesting to use `super(id = Random.rand(1..1000))` for to initialise `id` looks nice.